lint-staged.config.js

```diff
@@ -26,4 +26,7 @@ module.exports = {
     // For that reason, we move the `--onlyChanged` flag next to it.
     'yarn lint:css --reporters=jest-silent-reporter --onlyChanged',
   ],
+  'packages/application-config/schema.json': [
+    'bash -c "yarn --cwd packages/application-config build:schema"',
```

Comment:
Because otherwise the command will receive the file path from lint-staged. In this case, we just need to run a command, independently from the file.

It's the recommended approach from lint-staged if I remember.